cmd/lncli/commands.go

```diff
@@ -1977,6 +1977,19 @@ var listChainTxnsCommand = cli.Command{
 				"transactions until the chain tip, including " +
 				"unconfirmed, set this value to -1",
 		},
+		cli.UintFlag{
+			Name: "index_offset",
+			Usage: "The index of a transaction that will be " +
```

Comment:
nit: looks like we're keeping the descriptions lower case. So should start with `the index of ...` to keep things in line with other flags.